tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/nav-links-validator.js

```diff
@@ -0,0 +1,14 @@
+import ValidateContent from "../../../helpers/content-validators/content-validator.js";
+import { dataLoaded } from "../dpv-helpers/data-loaded-check.js";
+
+export const validateNavigationLinks = (dataMapper) => {
+    if (!dataLoaded(dataMapper.contents)) {
+        return;
+    }
+
+    const navigationLinks = Array.from(dataMapper.contents.get("navigationLink").entries());
```

Comment:
Lines 9 and 11 could be simplified:

```suggestion
    for (const navigationLink of dataMapper.contents.get("navigationLink").values()) {
        ValidateContent(navigationLink);
    }
```